mantra.md

```diff
@@ -22,10 +22,43 @@
 8. ACK Triangle
 ===============
 
+Contrary to belief of many new-age engineering shops; the utmost elegant
+products must be built in a environment constricted by bureaucratic hurdles.
+Not only does this does it prove that the product is fit enough for survival
```

Comment:
this is not english.

Do you meant:

"Not only this proves that the product is fit enough for survival for years to come, but also serves as ego re-enforcement of the involved fiefdoms"